torch/tensor.py

```diff
@@ -37,12 +37,16 @@ def __reduce_ex__(self, proto):
         return (torch._utils._rebuild_tensor_v2, args)
 
     def __setstate__(self, state):
-        if len(state) == 5:
```

Comment:
This method is only used in legacy serialization anyway, right?

Reply:
Yes, we now use `rebuild_tensor` and `rebuild_tensor_v2`. This is for loading super old check points that use `https://github.com/pytorch/pytorch/blob/v0.1.10/torch/tensor.py#L107`.